services/uptimeobserver/uptimeobserver-base.js

```diff
@@ -0,0 +1,59 @@
+import Joi from 'joi'
+import { BaseJsonService, InvalidParameter, InvalidResponse } from '../index.js'
+
+const errorResponse = Joi.object({
+  status: Joi.string().required(),
+  error: Joi.string(),
```

Comment:
Correct me if I'm wrong, but it seems the `status` field is never used when we have an error.
Let's remove it to save up on resources and avoid invalid response for info we do not use.

Also shouldn't the error field be required here? Otherwise the logic to pick up the error wouldn't work.

Reply:
You are right. Fixed